internal/command/terraform.tfstate

```diff
@@ -0,0 +1 @@
+{"version":4,"terraform_version":"1.9.0","serial":0,"lineage":"fake-for-testing","outputs":{"foo":{"value":"bar","type":"string","sensitive":true}},"resources":[],"check_results":null}
```

Comment:
Looks like this was accidentally committed.

Reply:
Oh how embarrassing, thanks.